### PR TITLE
Added link to `Transitive Persistence`

### PR DESCRIPTION
**Pull request by malukenho**
Linking #2190

/cc @lcobucci @Ocramius

### CODE REVIEW

File: docs/en/reference/working-with-objects.rst [hunk 1]
```diff
@@ -238,7 +238,7 @@ as follows:
    persist operation. However, the persist operation is cascaded to
    entities referenced by X, if the relationships from X to these
    other entities are mapped with cascade=PERSIST or cascade=ALL (see
-   "Transitive Persistence").
+   ":doc:`Transitive Persistence <working-with-associations>`").
```
Comment (lcobucci) [inline docs/en/reference/working-with-objects.rst]: Will this link to the correct section?

**Review by lcobucci — CHANGES_REQUESTED** [review]
@malukenho thanks, can you please also change the other occurrences of `see "Transitive Persistence"` in this document?

**Comment by malukenho** [comment]
@lcobucci done

**Comment by Ocramius** [comment]
👍

**Pull request merged**